src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/percent_widget.html.php

```diff
@@ -1 +1,13 @@
-<?php echo $view['form']->block($form, 'form_widget_simple', ['type' => isset($type) ? $type : 'text']) ?> %
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
```

Comment:
This should be reverted. The file would then look like this:

```php
<?php $symbol = false !== $symbol ? ($symbol ? ' '.$symbol : ' %') : '' ?>
<?php echo $view['form']->block($form, 'form_widget_simple', ['type' => isset($type) ? $type : 'text']).$symbol ?>
```

Reply:
Done, but fabbot.io is complaining about the license header 🤔

Reply:
Like anybody else, fabbot is not always right :)